### ISSUE TITLE
Public API for trade data

### ISSUE THREAD

**Issue by felivix** [aktionariat/reports#14]
Can we provide others a public API for trade data, like we have in the trades table widget?

**Comment by pansay** [comment]
the trades data api is: https://api.aktionariat.com/trades?ticker=VEDA

**Comment by felivix** [comment]
Thanks

**Comment by felivix** [comment]
Can you please have a look at this document and let me know if we can provide these API data to Nomics. I would like to get our VEDA brokerbot listed as an exchange on Nomics.

https://github.com/nomics-crypto/nomics-platform/blob/master/doc/cryptocurrency-api-exchange-integration.md

**Comment by Nicola8038** [comment]
Luzius, could you have a look at the document and tell Felix if he can provide this information to nomics-crypto?

**Comment by luziusmeisser** [comment]
@Nicola8038 : this is not about a one-time provision of data, it is about providing an API for price data for all tokens of our clients. So it is more work than just filling in a form.

**Comment by Nicola8038** [comment]
right, sorry didn't look at it closely enough. Who will look into this?

**Comment by luziusmeisser** [comment]
@bladekp What do you think would be the effort to implement the info and the markets endpoints?

**Comment by bladekp** [comment]
I do not think this is hard task, as far as I see there are 3 not complicated endpoints to implement on our end, the `info`, `markets` and `trades`, the rest are not relevant in our case or discouraged, it may take 2 maybe 3 working days with testing the endpoints using software they provide.

**Comment by Nicola8038** [comment]
Thanks for your feedback, Piotr. I would say implementation early March
should be the goal.

Best

--

Nicola Plain

CEO (ad interim)





Aktionariat AG

CHE-453.438.437

Mobile: +41 78 614 83 47

Email: ***@***.***

Web: https://aktionariat.com <http://aktionariat.com>



<https://t.sidekickopen06-eu1.com/s3t/c/5/f18dQhb0S7kC8fD6XKW5dK2qR2zGCz1N5xLs7slrBrYW1S-5-y3djZr_N5F2kFW3Zpn7f1CyYXb02?te=W3R5hFj4cm2zwW4mKLS-3ZWVWBW3K2-zv1JxwY5W1Lw3rh45LM6yW1Lvd7b3T1MdDW49M7Pd1Gy6MB0&si=8000000020887941&pi=8130eef7-b4aa-479e-a21f-535e0bfac25b>

<https://t.sidekickopen06-eu1.com/s3t/c/5/f18dQhb0S7kC8fD6XKW5dK2qR2zGCz1N5xLs7slrBrYW1S-5-y3djZr_N5F2kFW3Zpn7f1CyYXb02?te=W3R5hFj4cm2zwW4fQ47l4fGCmnW3Fbt5S3zcKScF43TBFPWrCS1&si=8000000020887941&pi=8130eef7-b4aa-479e-a21f-535e0bfac25b>

<https://t.sidekickopen06-eu1.com/s3t/c/5/f18dQhb0S7kC8fD6XKW5dK2qR2zGCz1N5xLs7slrBrYW1S-5-y3djZr_N5F2kFW3Zpn7f1CyYXb02?te=W3R5hFj4cm2zwW4mKLS-4rCvX6W4hCVj23Fbt5SW3F7xMK41RjQSW36dwWw363mD8W3VWt0F4pmP_4W4fxxF_3C36GB0&si=8000000020887941&pi=8130eef7-b4aa-479e-a21f-535e0bfac25b>

<https://t.sidekickopen06-eu1.com/s3t/c/5/f18dQhb0S7kC8fD6XKW5dK2qR2zGCz1N5xLs7slrBrYW1S-5-y3djZr_N5F2kFW3Zpn7f1CyYXb02?te=W3R5hFj4cm2zwW4fdJS31LdWZ2W3T1MdD49M7Pd0&si=8000000020887941&pi=8130eef7-b4aa-479e-a21f-535e0bfac25b>


*Aktionariat AG is a company founded by Murat Ögat & Luzius Meisser, a
Swiss Blockchain and Crypto Currency Pioneer. *

*Interested in blockchain? This article could be interesting for you: *Eine
Chance für den Finanzplatz | NZZ
<https://t.sidekickopen06-eu1.com/s3t/c/5/f18dQhb0S7kC8fD6XKW5dK2qR2zGCz1N5xLs7slrBrYW1S-5-y3djZr_N5F2kFW3Zpn7f1CyYXb02?te=W3R5hFj4cm2zwW4mKLS-41_40qW3F7smG3K53ntW41S2r143TbcQW41Yx_n3JFWPWW43P1h_3QZmG8W1GzPSM3JF3YjW3zdYNr1GBdn5W49h9zp41n_LcW41PG6b45RjbMW4t3zhP1J7CLKW22WLHc49Kv7ZW3F6hB74fN0QQ0&si=8000000020887941&pi=8130eef7-b4aa-479e-a21f-535e0bfac25b>

**Comment by felivix** [comment]
Any updates?

**Comment by felivix** [comment]
Any updates?

**Comment by KostasZav** [comment]
Hi @felivix. Apologies for the late reply.

Maybe a couple of words from my side.

- A swiss bank is currently working on a project to launch a swiss asset tokens aggregation platform and they have integrated the tokens of all our clients (incl. VEDA) with live data based on some endpoints we created for them. According to their feedback, they expect to go live by the end of October. We will communicate it once this is live.
- We have submitted an application to list the tokens of all our client at coingecko and waiting for their feedback
- We have been in contact with stomarket and they listed the tokens of our clients on their platform but there is a data mismatch. We got in touch to inform them about this issue but we haven't received feedback yet. We will follow up on that again
- We are currently checking the listing requirements from coinmarketcap in order to submit a DEX application soon.

What we can do is check if some of the endpoints already created for other projects would be sufficient for Nomics and give you an update. Does this work for you?
